### PR TITLE
Poke filter component redesign

### PR DESCRIPTION
**Pull request by philbu**
Some minor fixes and redesign of the pokemon filter tab of our popover.

I played a little bit with the design and made it more compact.
Do you like it? Do you have other design ideas? 
@PokemonGoers/catch-em-all 

selection_002
selection_003
selection_004
selection_005

### CODE REVIEW

**Comment by goldbergtatyana** [comment]
I like everything about it. Other opionions?

File: ionic2/app/components/poke-filter-pokemon-tab/poke-filter-pokemon-tab.component.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.background{
+  background-color: #f4f4f7;
+}
```
Comment (AlexanderLill) [inline ionic2/app/components/poke-filter-pokemon-tab/poke-filter-pokemon-tab.component.scss:3]: An empty line in between wouldn't hurt :)

File: ionic2/app/services/poke-types.ts [hunk 2]
```diff
@@ -1,20 +1,20 @@
 export const TYPES : { [key:string]:string; } = {
```
Comment (AlexanderLill) [inline ionic2/app/services/poke-types.ts:1]: Sorting those seems unnecessary to me, but okay

File: ionic2/app/components/poke-filter-pokemon-tab/poke-filter-pokemon-tab.component.ts [hunk 3]
```diff
@@ -54,7 +57,7 @@ export class PokeFilterPokemonTabComponent implements OnInit {
 
     for (let str in this.apiservice.getTypes()) {
```
Comment (AlexanderLill) [inline ionic2/app/components/poke-filter-pokemon-tab/poke-filter-pokemon-tab.component.ts]: While you are at it anyway: Can you rename the `apiservice` to `apiService`? :)

**Review by AlexanderLill — CHANGES_REQUESTED** [review]
I like the design very much :)

File: ionic2/app/components/poke-filter-pokemon-tab/poke-filter-pokemon-tab.component.scss (hunk 1 above)
Comment (philbu) [reply ionic2/app/components/poke-filter-pokemon-tab/poke-filter-pokemon-tab.component.scss:3]: Done

File: ionic2/app/services/poke-types.ts (hunk 2 above)
Comment (philbu) [reply ionic2/app/services/poke-types.ts:1]: It's not. Your pokemon-filter is displaying the types without sorting them

File: ionic2/app/components/poke-filter-pokemon-tab/poke-filter-pokemon-tab.component.ts (hunk 3 above)
Comment (philbu) [reply ionic2/app/components/poke-filter-pokemon-tab/poke-filter-pokemon-tab.component.ts]: Done :)

**Comment by philbu** [comment]
@AlexanderLill can you approve my changes?

**Comment by MajorBreakfast** [comment]
I like the design. Good work!

The behavior of the pokemon filter needs some adjustment. Checking the boxes never does what I expect. The intended behavior is specified in https://github.com/PokemonGoers/Catch-em-all/issues/74 I can work on that as soon as this is merged

Feel free to merge this. I see no issues.

**Comment by WoH** [comment]
@philbu Did you maybe forget something? :laughing:

**Comment by MajorBreakfast** [comment]
That's why I always use my own repo. It currently has 8 branches and no one cares.

**Comment by johartl** [comment]
@philbu Can this branch be deleted?

**Comment by MajorBreakfast** [comment]
@johartl Polite way for saying "You forgot to delete the branch!" :)

**Comment by johartl** [comment]
Always trying to be polite ;)

**Pull request merged**